code/modules/weather/weather_fsm_states.dm

```diff
@@ -199,12 +207,37 @@
 		"The wind howls around you.",
 		"Swirling sand obscures your vision."
 	)
-	protected_messages = list("Stinging sand blows against $ITEM$.")
+	protected_messages = list("$ITEM$ shields you from the howling sandstorm.")
 	ambient_sounds = list('sound/effects/weather/sandstorm.ogg')
 
 /singleton/state/weather/sandstorm/handle_exposure_effects(mob/living/M, obj/abstract/weather_system/weather)
-	to_chat(M, SPAN_DANGER("You are blasted by a gust of stinging sand!"))
-	M.adjustBruteLoss(rand(1,5))
+	if((istype(M, /mob/living/simple_animal) && M.faction == "Moghes") || M.get_species(FALSE) == SPECIES_MONKEY_UNATHI) //Not realistic that they're just immune, but we don't have AI for simplemobs seeking shelter from storms.
+		return
+	else if(isvaurca(M)) //Bugs have sealed carapaces
+		to_chat(M, SPAN_WARNING("Your carapace protects you from the stinging sand!"))
+	else if(isipc(M) || issilicon(M)) //Metal is more durable than meat
+		to_chat(M, SPAN_DANGER("Your chassis is scratched by a gust of stinging sand!"))
+		M.apply_damage(1, DAMAGE_BRUTE)
```

Comment:
for synthetics this is gonna be a pain in the ass because their damage doesn't heal

i don't think this is necessarily a bad thing but you should keep that in mind

Reply:
Given the number of new protective options available for weather, I don't think this will be a severe issue, but we'll have to see how it plays out